app/dashboard/models.py

```diff
@@ -4896,3 +4896,18 @@ class TribeMember(SuperModel):
         max_length=20,
         blank=True
     )
+
+    @property
+    def mutual_follow(self):
+        return TribeMember.objects.filter(profile=self.org, org=self.profile).exists()
+
+    @property
+    def mutual_follower(self):
+        tribe_following = TribeMember.objects.filter(profile=self.profile).values('org')
+        return TribeMember.objects.filter(org=self.org, profile__in=tribe_following).exclude(profile=self.profile)
+
+    @property
+    def mutual_following(self):
+        tribe_following = TribeMember.objects.filter(org=self.profile).values('profile')
+        print(tribe_following)
+        return TribeMember.objects.filter(org__in=tribe_following, profile=self.org)
```

Comment:
not sure about this logic, if you already have those who already following you why not better cross both and have the result to avoid many queries?
Also I see a missing print there

Reply:
Whats the missing print? Is there a rule for printing objects here we have to follow?

Reply:
done, only one query is executed on each of these methods using a subquery